f/connectors/alerts/alerts_gcs.py

```diff
@@ -124,221 +131,251 @@ def _get_tif_rel_filepath(blob_name, local_file_path, territory_id):
     Returns
     -------
     str
-        The relative file path for the TIF file.
+        The relative file path for the file.
     """
-    year = local_file_path.split("/")[-3]
-    month = local_file_path.split("/")[-2]
-    filename = local_file_path.split("/")[-1]
+    path_parts = Path(local_file_path).parts
+    year = path_parts[-3]
+    month = path_parts[-2]
+    filename = path_parts[-1]
     alert_id = filename.split(".")[0].split("_")[-1]
-    filepath = os.path.join(str(territory_id), year, month, alert_id, "images")
-    return filepath
+    filepath = Path(str(territory_id), year, month, alert_id)
+    if filename.lower().endswith((".tif", ".tiff", ".jpg", ".jpeg")):
+        filepath = filepath / "images"
+    return str(filepath)
 
 
 def sync_gcs_to_local(
-    dst_path, storage_client, bucket_name, territory_id, alerts_metadata_filename
+    destination_path,
+    storage_client,
+    bucket_name,
+    territory_id,
+    alerts_metadata_filename,
 ):
     """Download files from a GCS bucket to a local directory.
 
     Parameters
     ----------
-    dst_path : str
+    destination_path : str
```

Comment:
I see you coercing `destination_path` (str) to a Path many times in this function.  You could do this once at the very start.


You could even make the function a tad more flexible to accept either str or Path; I see this done in many other libraries.  Example:

```py
def sync_gcs_to_local(destination_path):
     """
    Parameters
    ------------
    destination_path : str or pathlib.Path
        The path to the destination. Can be a string or a pathlib.Path object.
    """
    destination_path = Path(destination_path)
    ...
```

This works because the Path constructor can take either a string or another Path object as an argument. If you pass another Path object, it will simply return that object unchanged.